rasa_nlu/project.py

```diff
@@ -131,14 +131,20 @@ def unload(self, model_name):
         return unloaded_model
 
     def _latest_project_model(self):
-        """Retrieves the latest trained model for an project"""
+        """Retrieves the latest trained model for a project"""
+
+        models = {}
+        for model in self._models.keys():
+            try:
+                key = self._read_model_metadata(model).metadata.get("trained_at")
```

Comment:
this won't work for models stored in the cloud as `_read_model_metadata` downloads the whole model. hence this will download all models for every requested project (instead of only downloading the latest / requested model).

One way to fix this: Make sure that the trained timestamp is part of the uploaded tar file name when uploading a model to a remote model storage server.